client/components/profilePage/allUsersModal.jsx

```diff
@@ -28,14 +32,19 @@ export default function AllUsers({
           <Col>
             {allUsers.map((user) => {
               return (
-              <div key={user._id}>
-                <span className='thumbBody'><img className='thumbImage' src = {user.photoUrl}/></span>
+              <Row className = 'm-3' key={user._id}>
```

Comment:
The spinner will never be used because allusers is initialized with an empty array, which will never be undefined. Consider refactoring to use [show, setShow] state variable to determine whether the spinner or the modal displays or something. 

Do later!